miner/worker.go

```diff
@@ -186,10 +181,10 @@ type worker struct {
 	resubmitHook func(time.Duration, time.Duration) // Method to call upon updating resubmitting interval.
 
 	// Needed for randomness
-	db *ethdb.Database
+	db ethdb.Database
```

Comment:
For my own curiosity, why did this switch from a pointer to a literal?

Reply:
`ethdb.Database` is actually an interface type where the concrete value of it is a pointer.  So `db` was originally a pointer to a pointer.

Also, all other uses throughout the gocelo code uses the `ethdb.Database` type as a literal.

Reply:
BTW, the `db` variable was one that clabs originally added.

Reply:
Makes sense